src/data/scenarioOutline.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+skinparam activity {
+BackgroundColor #76CDD8
+ArrowColor black
+ArrowFontSize 13
+ArrowFontStyle bold
+ArrowFontName Segoe UI
+BorderColor black
+FontName Segoe UI
+FontSize 15
+}
+(*) --> "handle Scenario Outline"
+-right-> "onScenarioOutline"
+-left-> "handle Scenario Outline"
+--> "[*Tags]"
+--> "filter/pre Step"
+-right-> "pre Step"
+-left-> "filter/pre Step"
+--> "Process steps"
+If "Are there unprocessed steps?"
+-left-> [Yes] "Process steps"
```

Comment:
I don't think that this "Process steps" part is correct, given that you have the actual processing part in the "No" branch. Same for the "Process examples"